CHANGELOG.unreleased.md

```diff
@@ -1081,6 +1081,39 @@ sometimes{{nogaps}}areimportant
 {{name}} is your name
 ```
 
+#### JavaScript: Break array expression in jest each function parameters([#6694] by [@sosukesuzuki])
+
+<!-- prettier-ignore -->
+```js
+// Input
+test.each([
+  { a: "1", b: 1 },
+  { a: "2", b: 2 },
+  { a: "3", b: 3 },
+])("test", ({ a, b }) => {
+    expect(Number(a)).toBe(b);
+  }
+);
+
+// Output (Prettier stable)
+test.each([{ a: "1", b: 1 }, { a: "2", b: 2 }, { a: "3", b: 3 }])(
+  "test",
+  ({ a, b }) => {
+    expect(Number(a)).toBe(b);
+  }
+);
+
+// Output (Prettier master)
+test.each([
+  { a: "1", b: 1 },
+  { a: "2", b: 2 },
+  { a: "3", b: 3 },
+])("test", ({ a, b }) => {
+    expect(Number(a)).toBe(b);
+  }
+);
```

Comment:
Did you accidentally copy the input, rather than the output? The indentation looks wrong here.

Reply:
Thanks, I fixed:+1: